src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1225,11 +1219,9 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
             TargetFramework targetFramework = TargetFramework.Standard,
             string assemblyName = "",
             string sourceFileName = "",
-            bool skipUsesIsNullable = false,
-            RuntimeFlag runtimeFeature = RuntimeFlag.None)
+            bool skipUsesIsNullable = false)
         {
-            Debug.Assert(targetFramework == TargetFramework.Standard || runtimeFeature == RuntimeFlag.None);
-            if (runtimeFeature == RuntimeFlag.ByRefFields)
+            if (targetFramework == TargetFramework.Net70)
```

Comment:
>if (targetFramework == TargetFramework.Net70) [](http://example.com/codeflow?start=12&length=45)

I actually thought that we would add a set of corresponding ref assemblies, perhaps fake, rather that keeping any special cases in CreateCompilation. This implementation feels fragile because one has to remember removing/adjusting it once real Net70 ref assemblies are added. Otherwise this helpert will keep hijacking references. I actually this is worse than having the ```runtimeFeature``` parameter. Given this, at the moment, I would prefer removing this if completely and adding real Net70 ref assemblies.

Reply:
Let's discuss after next standup.